### PR TITLE
Add Beheading, Lifesteal, and update PT-BR translations.

### PR DESCRIPTION
**Pull request by vini2003**

### CODE REVIEW

**Review by Shnupbups — APPROVED** [review]

**Review by Juuxel — CHANGES_REQUESTED** [review]
Could you run the added classes through a reformatter? Some of them have a mix of tabs and spaces as indentation.

File: src/main/java/com/fabriccommunity/spookytime/enchantment/BeheadingEnchantment.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	public static boolean hasBeheading(LivingEntity livingEntity) {
+    	return EnchantmentHelper.getEquipmentLevel(SpookyEnchantments.BEHEADING, livingEntity) > 0;
+    }
+
+	public static boolean getHead(DamageSource damageSource) {
+    	if (damageSource.getAttacker() instanceof LivingEntity && hasBeheading((LivingEntity)damageSource.getAttacker())) {
+			LivingEntity attacker = (LivingEntity)damageSource.getAttacker();
+			int enchantmentLevel = EnchantmentHelper.getEquipmentLevel(SpookyEnchantments.BEHEADING, attacker);
+			int scytheMultiplier = attacker.getActiveItem().getItem() == SpookyItems.REAPERS_SCYTHE ? 2 : 1;
+			return new Random().nextInt(100) <= dropChance.get(enchantmentLevel - 1) * scytheMultiplier;
```
Comment (Juuxel) [inline src/main/java/com/fabriccommunity/spookytime/enchantment/BeheadingEnchantment.java]: I think you can use the world's `Random` here.

File: src/main/java/com/fabriccommunity/spookytime/enchantment/LifestealEnchantment.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+import net.minecraft.enchantment.Enchantment;
+import net.minecraft.enchantment.EnchantmentHelper;
+import net.minecraft.enchantment.EnchantmentTarget;
+import net.minecraft.entity.EquipmentSlot;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.entity.damage.DamageSource;
+
+public class LifestealEnchantment extends Enchantment {
+	protected static final ImmutableList<Float> stealMultiplier = ImmutableList.of(0.1F, 0.12F, 0.15F, 0.17F, 0.2F);
```
Comment (Juuxel) [inline src/main/java/com/fabriccommunity/spookytime/enchantment/LifestealEnchantment.java]: This should be `STEAL_MULTIPLIER` as it's a static final (and immutable) field.

**Review by Juuxel — CHANGES_REQUESTED** [review]

**Review by Shrimpstress — CHANGES_REQUESTED** [review]
Please fix the formatting. There's weird mixed usage of 4 spaces and tabs in multiple classes. Otherwise everything looks great.

**Comment by vini2003** [comment]
[Will fix, blame IDEA. It all looks fine in it.](https://i.imgur.com/aFmfGUs.png)

**Review by Shrimpstress — APPROVED** [review]

**Review by Juuxel — APPROVED** [review]

**Pull request merged**